runners/apex/src/main/java/org/apache/beam/runners/apex/translation/operators/ApexParDoOperator.java

```diff
@@ -83,6 +94,12 @@
   @Bind(JavaSerializer.class)
   private final List<PCollectionView<?>> sideInputs;
 
+  private StateInternalsProxy<?> currentKeyStateInternals;
```

Comment:
Seems like this variable is never assigned to, so all branches testing for it being non-null are currently dead code. Suppose that's coming in a followup PR?